app/controllers/records_controller.rb

```diff
@@ -1,6 +1,22 @@
 class RecordsController < ApplicationController
   include RecordsControllerBehavior
 
+  def choose_ingest_options
+    @templates = [["Raw (no template)", nil]] + FormTemplate.all.collect {|t| [t.title, t.id]}
+  end
+
+  def new
+    @form = build_form
+    find_template
+    render 'records/new'
```

Comment:
Hydra Editor's create and update methods directly call new and edit, so the render has to happen that way otherwise you'll get "Missing template records/create, application/create".

However, in testing this I discovered multiple bugs!  So don't merge even if I address everything else.